pom.xml

```diff
@@ -52,6 +52,12 @@
         <version>${nuxeo.dam.nxr.version}</version>
         <type>zip</type>
       </dependency>
+      <dependency>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>nuxeo-platform-3d-core-dependencies</artifactId>
+        <type>pom</type>
+        <version>${nuxeo.distribution.version}</version>
+      </dependency>
```

Comment:
Dependency management for `nuxeo-platform-3d-core-dependencies` should not be here but in the `addons/pom.xml` like other `nuxeo-platform-3d-*` dependencies.
https://github.com/nuxeo/nuxeo/blob/10.10/addons/pom.xml#L1056